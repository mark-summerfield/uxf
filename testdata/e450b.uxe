uxf 1
(int)
