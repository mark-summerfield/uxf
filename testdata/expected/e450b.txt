uxf.py:error:uxf:F450:e450b.uxe:1:expected table ttype, got int
